src/components/LinkMenu.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import { EditorView } from "prosemirror-view";
+import LinkEditor, { SearchResult } from "./LinkEditor";
+import FloatingToolbar from "./FloatingToolbar";
```

Comment:
reusing display logic in `FloatingToolbar`

Reply:
Selection of "meats" appreciated